### DISCUSSION TITLE
Cannot import 3rd party css out of node_modules

### DISCUSSION THREAD

**Discussion by congwenma** [vercel/next.js#25133] [Help]
### What version of Next.js are you using?

10.1.3

### What version of Node.js are you using?

14.x

### What browser are you using?

Chrome

### What operating system are you using?

MacOS

### How are you deploying your application?

next start

### Describe the Bug

I could not import a 3rd party css file via the recommended approach listed on https://nextjs.org/docs/basic-features/built-in-css-support#import-styles-from-node_modules. 
notably this isn't currently supported.

> For importing CSS required by a third party component, you can do so in your component. For example:

The attempt would throw out this error:

> Global CSS cannot be imported from files other than your Custom

It seems there were a few threads opened requesting the same feature to be implemented / allowed in next.js, have these been addressed?

https://github.com/vercel/next.js/issues/10059#issuecomment-607444285

### Expected Behavior

node_modules stylesheet should be encapsulated in the component without this warning, next.js would only bundle the css for pages that have the component imported.

### To Reproduce

1. create an `nx` project with `next.js`
2. import a third party library
3. import its css inside a non global app.tsx file

**Comment by timneutkens** [comment]
Opened a RFC here: #27953
